### ISSUE TITLE
Data Tracker > Moped Migration planning

### ISSUE THREAD

**Issue by zappfyllc** [cityofaustin/atd-data-tech#7775]
[Working doc](https://docs.google.com/document/d/1JPs4n25y_WKATexcI4vmUjgNdqQZu9cO70uxXYq5als/edit).

**Comment by amenity** [comment]
@zappfyllc @johnclary - who were you thinking should lead this?

**Comment by zappfyllc** [comment]
@amenity @johnclary I can prioritize this for myself for this sprint but I might not get to it until after Backlog Grooming, so happy to pass it to someone else if they have availability to tackle this sooner. Mainly, I think we need to answer the following questions:

1. How are we dealing with data already in Moped staging/production?

- If we delete all data in Moped staging/production, we need to specify what tables for the script (e.g. `moped_projects`, `moped_project_files`, etc.
- When does this deletion occur? How do we handle test data from AMD users in the onboarding process and do we need to repeat the deletion?

2. How do we migrate projects and relevant project-related data from Data Tracker to Moped?

- What do we need in terms of Knack IDs, project IDs, etc.?
- How do we handle relational data that might be stored in Knack columns as comma-separated values or strings coming from multiple choice values? We need a step-by-step breakdown of what rows to create in what tables in Moped since not all data is flattened to the `moped_projects` table but rather stored in related tables (e.g. `moped_project_files`, `moped_proj_notes`, etc.)

3. How do we migrate signals from Data Tracker to Moped components?

- I'm actually unsure about the mapping dependencies to ask the appropriate questions: maybe question 2 (see above) will handle a lot of the answer to this question, but just noting that we should be explicit in how we're storing JSONB and other project signal component data.

**Comment by amenity** [comment]
Whether or not we fit #7993 into our 1.0 release, the migration would be a good time to give all the existing projects standardized milestones and phases.

**Comment by amenity** [comment]
@johnclary will schedule time to review; essentially importing a CSV to Moped.

**Comment by johnclary** [comment]
@amenity @zappfyllc  @sergiogcx I renamed this issue and converted it to an epic. Started [this doc](https://docs.google.com/document/d/1JPs4n25y_WKATexcI4vmUjgNdqQZu9cO70uxXYq5als/edit) for organizing thoughts. 

I made progress on https://github.com/cityofaustin/atd-data-tech/issues/8067. I have the components sorted out at least. I blocked off 1/14 as a day to focus on wrapping up the bulk of that work. Will hopefully have time before then to keep moving it forward.

The tasks of refreshing production and cleaning up staging users are important can be done sooner. I did not create issues.
